test/e2e/govmomi_test.go

```diff
@@ -89,23 +73,31 @@ func destroyVMsWithPrefix(prefix string) {
 	}
 }
 
-// nolint:deadcode
+// nolint:deadcode,unused
```

Comment:
Can we just remove this if it is unused? Or is it used and just not detected by the linter?

Reply:
my preference is too delete too, wasn't sure why it was marked like that in the first place. Gonna do it anyway I think